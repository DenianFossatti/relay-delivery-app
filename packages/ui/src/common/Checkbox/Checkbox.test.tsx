import { mockTestID, render, wrapper } from '../../tests';

import { Checkbox, CheckboxProps } from '.';

describe(`Checkbox`, () => {
  const props: CheckboxProps = {
    testID: 'Checkbox',
    value: 'value',
  };

  const { findByTestId, toJSON } = render(<Checkbox {...props} />, { wrapper });

  const el_container = mockTestID('View', props.testID!);

  it(`render component #${el_container}`, async () => {
    const sut = await findByTestId(el_container);
    expect(sut).toBeTruthy();
  });

  it('should render snapshot', () => {
    expect(toJSON()).toMatchSnapshot();
  });
});
