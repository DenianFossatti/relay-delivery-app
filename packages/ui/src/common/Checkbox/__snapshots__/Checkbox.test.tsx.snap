// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms/Checkbox should render snapshot 1`] = `null`;
